lib/passport/passport.js

```diff
@@ -0,0 +1,84 @@
+const cookie_parser = require('cookie-parser');
+const body_parser = require('body-parser');
+const session = require('express-session');
+const passport = require('passport');
+const local_strategy = require('passport-local').Strategy;
+
+// temp user account
+const acc1 = {
+    id: 1,
+    username: 'test',
+    password: 'test',
+    security: 10
+}
+
+// temp admin account
+
+const acc2 = {
+    id: 2,
+    username: 'admin',
+    password: 'admin',
+    security: 1
+}
+
+const accs = [acc1, acc2]
+
+module.exports = (router) => {
+    router.use(cookie_parser());
+    router.use(body_parser.urlencoded({ extended: false }));
+    router.use(session({ secret: 'cats' }));
```

Comment:
This should be changed, and not hard coded. Consider using an environment variable that is set on server start